docs/transparent-validation/sd-wire/usage-validation.md

```diff
@@ -116,80 +197,44 @@ Dev: 0, Manufacturer: SRPOL, Serial: sd-wire_11, Description: sd-wire
 
-### Scenario using SDWire and RTE
+## SDWire with RTE use scenario
 
 1. Insert SD card to the SDWire.
-1. Insert SDWire into the DUT and connect it to the TS with micro-USB-->USB
+2. Insert SDWire into the DUT and connect it to the TS with micro-USB-->USB
    cable.
-1. Connect [RTE][shop1] power control connectors to the DUT (RTE here is
+3. Connect [RTE][shop1] power control connectors to the DUT (RTE here is
    optional, it helps with automated and remote power control of the connected
    device).
-1. Check serial no. of SDWire:
-```
-sudo  sd-mux-ctrl --list
-```
-Output:
-```
-Number of FTDI devices found: 1
-Dev: 0, Manufacturer: SRPOL, Serial: sd-wire_11, Description: sd-wire
-```
-1. Disconnect power supply using RTE.
-1. Connect SD card to the TS (using sd-mux-ctrl)
-```
-sudo sd-mux-ctrl --device-serial=sd-wire_11 --ts
-```
-1. Flash the SD card usign `bmaptool` or balenaEtcher.
-1. Connect SD card to the DUT (using sd-mux-ctrl)
-```
-sudo sd-mux-ctrl --device-serial=sd-wire_11 --ts
-```
-1. Connect power supply using RTE.
-1. DUT should boot from freshly burned SD card.
-
-> Command `sudo sd-mux-ctrl --device-serial=sd-wire_11 --status` returns
-information if SDWire is connected to DUT or TS.
-```
-➜  ~ sudo sd-mux-ctrl --device-serial=sd-wire_11 --status
-SD connected to: TS
-```
-At the moment RTE does not support sd-mux-ctrl, so SDWire must be controlled
-from configured TS (host computer).
-
-## Presale validation
-
-Hardware requirements:
-
-* SDWire
-* SD card
-* DUT bootable from SD card (RPI, Orange PI etc.)
-* DUT power supply
-* Micro-USB-->USB cable
-
-Validation steps:
-
-1. Prepare environment in accordance with this [section](#evnironment-preparation).
-1. Insert SD card to the SDWire.
-1. Put SDWire into the DUT and connect it to the TS with the micro USB - USB
-   cable.
-1. Check if SDWire is preconfigured:
-```
-sudo  sd-mux-ctrl --list
-Number of FTDI devices found: 1
-Dev: 0, Manufacturer: SRPOL, Serial: sd-wire_11, Description: sd-wire
-```
-If any FTDI device is found, it means that SDWire is preconfigured, if not - go
-to [first use](#first-use) section and prepare the device.
-1. Connect SD card to the TS:
-```
-sudo sd-mux-ctrl --device-serial=sd-wire_11 --ts
-```
-1. Flash the SD card usign `bmaptool` or balenaEtcher.
-1. Connect SD card to the DUT using `sd-mux-ctrl`:
-```
-sudo sd-mux-ctrl --device-serial=sd-wire_11 --DUT
-```
-1. Connect power supply to the DUT and check if it boots properly from newly
-   burned image.
+4. Check serial no. of SDWire:
+    ```
+    sudo  sd-mux-ctrl --list
+    ```
+    Output:
+    ```
+    Number of FTDI devices found: 1
+    Dev: 0, Manufacturer: SRPOL, Serial: sd-wire_11, Description: sd-wire
+    ```
+5. Disconnect power supply using RTE.
+6. Connect SD card to the TS (using sd-mux-ctrl)
+    ```
+    sudo sd-mux-ctrl --device-serial=sd-wire_11 --ts
+    ```
+7. Flash the SD card using `bmaptool` or balenaEtcher.
```

Comment:
same as above with download links and example command

Reply:
Description in the newest pushed version has been added.